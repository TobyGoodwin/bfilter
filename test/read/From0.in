From fakefrom
Dull: header

T24gTW9uLCBNYXkgMjYsIDIwMTQgYXQgMTI6MzAgUE0sIFRlcmVuY2UgUGFyciA8cGFycnRAY3Mu

From fakefrom2
Dull: heade

dXNmY2EuZWR1PiB3cm90ZToKPiBIaS4gSnVzdCBjdXJpb3VzLiBIYXMgYW55b25lIGZpZ3VyZWQg

that line was b64 encoded, but this one isn't
