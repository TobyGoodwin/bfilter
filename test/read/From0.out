{On Mon, May 26, 2014 at 12:30 PM, Terence Parr <parrt@cs.}
{usfca.edu> wrote:
> Hi. Just curious. Has anyone figured }
{that line was b64 encoded, but this one isn't}
